### ISSUE TITLE
[Feature Request] Multiple nametag group per user

### ISSUE THREAD

**Issue by KohGeek** [sgtcaze/NametagEdit#56]
It is possible that I assign one group for prefix and another group for suffix for a user?
That would be really nice as it will allow me to further customize my server.

However I realised you have said that it is not possible with the current scoreboard config, but it wasn't clear if separate group with only prefix and suffix will work.

**Comment by sgtcaze** [comment]
It's possible to have different Groups to specific Prefix/Suffixes, and have players belong to multiple groups (just needs a few tweaks). The only thing is how we prioritize which prefix/suffix a player will get from which group. I'd imagine there would be some overlap. Thoughts?

**Comment by KohGeek** [comment]
I don't quite understand the overlap part. Maybe you can add a priority key to it so that the top priority group will be shown?

I was wondering if it will benefit from splitting the prefix and suffix into two configs, so that when the plugin processes it, it needs to combine the two then initiate the scoreboard.

Sorry if I don't make much sense, just started programming in Java recently.

**Comment by sgtcaze** [comment]
If I am understanding you correctly, you want the following:

Group1:
  Prefix: MyPrefix
Group2:
  Suffix: MySuffix

Two groups where a player can have the prefix from one group, and suffix from another. Is this what you mean?

**Comment by KohGeek** [comment]
Yes indeed.

**Comment by KohGeek** [comment]
Yes as of now this is impossible and you code neatly prevent us to have two permissions in case of errors, which is good.

**Comment by sgtcaze** [comment]
I am in the PST time zone. Yeah the HashMaps that store group and player prefix/suffix data are pretty strict at the moment.
